configs/i18n.js

```diff
@@ -23,7 +24,8 @@ module.exports = {
         home: 'Accueil',
         more: "Plus d'infos",
         register: "S'inscrire",
-        slides: 'Voir les diapositives'
+        slides: 'Voir les diapositives',
+        speaker: 'Conférencier | Conférenciers'
```

Comment:
How can we deal with feminine in french (Ex: `Conférencière`)

Reply:
Very good point! I've just added a gender prop to the speaker object in storyblok. If all the speakers are feminine, it will now display the proper genre following the idea you gave here! 😃 

NB: I've added those 3 options for now - the goal being to display feminine when it applies: 

<img src=https://user-images.githubusercontent.com/1448836/89141650-31833a00-d513-11ea-9ea7-26b41ea501b9.png width="30%">

I've updated the the info for the feminine speaker we had until now.